mne/commands/mne_coreg.py

```diff
@@ -29,29 +29,39 @@ def run():
                       "the data source panel and the coregistration panel "
                       "into a single panel with tabs.")
     parser.add_option("--no-guess-mri", dest="guess_mri_subject",
-                      action='store_false', default=True,
+                      action='store_false', default=None,
                       help="Prevent the GUI from automatically guessing and "
                       "changing the MRI subject when a new head shape source "
                       "file is selected.")
-    parser.add_option("--head-opacity", type=float, default=1.,
+    parser.add_option("--head-opacity", type=float, default=None,
                       dest="head_opacity",
                       help="The opacity of the head surface, in the range "
                       "[0, 1].")
+    parser.add_option("--high-res-head",
+                      action='store_true', default=False, dest="high_res_head",
+                      help="Use a high-resolution head surface.")
     parser.add_option("--low-res-head",
                       action='store_true', default=False, dest="low_res_head",
-                      help="Default to using a low-resolution head surface.")
+                      help="Use a low-resolution head surface.")
     parser.add_option('--verbose', action='store_true', dest='verbose',
                       help='Turn on verbose mode.')
 
     options, args = parser.parse_args()
 
+    if options.low_res_head:
+        head_high_res = False
```

Comment:
I meant whether printing a reason and calling sys.exit() would be more appropriate since we're in a script and the conflicting arguments were specified in bash, not in a Python function call.